pytorch_lightning/strategies/deepspeed.py

```diff
@@ -447,6 +448,8 @@ def init_deepspeed(self):
         if self.lightning_module.trainer.gradient_clip_algorithm == GradClipAlgorithmType.VALUE:
             raise MisconfigurationException("DeepSpeed does not support clipping gradients by value.")
 
+        if isinstance(self.accelerator, CPUAccelerator):
```

Comment:
I think it could be done inside init? since we already have `accelerator` there.